src/annotator/integrations/test/html-metadata-test.js

```diff
@@ -405,5 +405,24 @@ describe('HTMLMetadata', () => {
 
       assert.equal(doc.uri(), canonicalLink.href);
     });
+
+
+    it('should log an error if URI is not decodable', () => {
+      // Arrange
+      // '%%CUST_ID%%' is an invalid escape sequence, so it will throw a URIError when decoded
+      const badURI = 'https://example.com/?foo=%%CUST_ID%%' 
+      sinon.stub(testDocument, '_getDocumentHref').returns(badURI); 
+      const consoleErrorSpy = sinon.spy(console, 'error');
+
+      // Act
+      testDocument.uri(); // The method that triggers the try...catch
+      
+      // Assert
+      assert(consoleErrorSpy.calledOnce);
+      assert(consoleErrorSpy.calledWith('Error decoding URI:', sinon.match.instanceOf(URIError)));
```

Comment:
We usually use the more specific forms of `assert`, because they read more clearly.

```suggestion
      assert.calledOnce(consoleErrorSpy);
      assert.calledWith(consoleErrorSpy, 'Error decoding URI:', sinon.match.instanceOf(URIError));
```